src/adapters/scene.js

```diff
@@ -23,6 +23,8 @@ const baseUrl = nconf.get().system.baseUrl;
 
 const store = new LocalStore();
 
+let longTouchTimeout = null;
```

Comment:
You can move it inside your `if(mobileDevice()) {}` block below. That way all the code related to the long touch is close and can be read and maintained more easily.